lib/util/cidr.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module Util
+
+  # provides helper methods for interacting with CIDR network addresses 
+  module CIDR
+    class << self
+
+      # returns the formatted CIDR formatted string for a network
+      def format_cidr cidr
+        cidr.is_a?(IPAddr) ? "#{cidr}/#{cidr_mask cidr}" : cidr.to_s
+      end
+
+      # returns the length of the netmask in bits
+      def cidr_mask cidr
+        mask = cidr.instance_variable_get(:@mask_addr).to_s(2)[/\A(1*)0*\z/, 1]
```

Comment:
Is Cidr an object you don't own?  Seems like this should just be a simple accessor call?

Reply:
It is not, it is the `IPAddr` class, but I did some digging and I have some ideas for how to clean this up a little.